QuTip Holstein-Tavis-Cummings

FrancisJaune opened the pull request:
The Holstein-Tavis-Cummings model built in QuTip.

FrancisJaune committed:
QuTip Holstein-Tavis-Cummings

The Holstein-Tavis-Cummings model built in QuTip.

FoleyLab committed:
added comment on Qutip_Holstein-Tavis-Cummings notebook

FoleyLab commented:
@FrancisJaune this looks good, the eigenvalues indeed match!   

I have one comment which I also added to your notebook and reproduce here:

Regarding lines:
>> q = np.array([[0, 0],[0, 1]], dtype=complex) # |e><e|
>> proj_e = Qobj(q)

Can you find a way to buld proj_e in terms of the built-in sigmap() and sigmam() functions?
Question/Hint: Can you explain what the proj_e operator does with respect to the basis states |g> and |e>
Similarly, what do the operators sigmap() and sigmam() do to the basis functions |g> and |e>?

FrancisJaune committed:
QuTip_Holstein_Tavis_Cummings_updated

QuTip implementation of the Holstein-Tavis-Cummings model updated to create proj_e using sigmap() and sigmam()

FrancisJaune commented:
I uploaded the updated QuTip_Holstein-Tavis-Cummings

FoleyLab committed:
did some organizing of vibronic folder
